src/dawn/CodeGen/CodeGen.cpp

```diff
@@ -66,8 +66,8 @@ void CodeGen::addTmpStorageInit_wrapper(MemberFunction& ctr,
                                         const std::vector<std::string>& tempFields) const {
   if(!(tempFields.empty())) {
     auto verticalExtent = getVerticalTmpHaloSizeForMultipleStencils(stencils);
-    ctr.addInit(bigWrapperMetadata_ + "(dom.isize(), dom.jsize(), dom.ksize() + 2*" +
-                std::to_string(verticalExtent) + ")");
+    ctr.addInit(bigWrapperMetadata_ + "(dom.isize(), dom.jsize(), dom.ksize() /*+ 2 *" +
```

Comment:
why adding this comment on the generated code?

Reply:
this would be an ok maximum extension of the domain locally. The problem that we're currently facing is that we cannot generate like that since the storages need to precisely match the passed storages form the GT side. once this changes we can actually generate properly